### PR TITLE
Update 0.FAQ.md

### PR DESCRIPTION
**Pull request by foesa-yang**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: docs-2.0/20.appendix/0.FAQ.md [hunk 1]
```diff
@@ -380,6 +380,23 @@ You also need to run [Balance Data and Balance leader](../8.service-tuning/load-
 
 Hosts with the status of `OFFLINE` will be automatically deleted after one day.
 
+### "How do I view the dmp file?"
+
+The dmp file is an error report file detailing the process exit information, which can be viewed with the gdb tool that comes with the operating system. The dmp file is saved in the current directory of the startup binary(default is `/usr/local/nebula` directory) and will be generated automatically when the NebulaGraph service crashes.
+1. Check the Core file process name, pid is usually a numeric value.
+   ```
+   $ file core.<pid>
```
Comment (whitewum) [inline docs-2.0/20.appendix/0.FAQ.md:388]: give an example

**Review by cooper-lzy — APPROVED** [review]

**Review by abby-cyber — APPROVED** [review]

**Comment by abby-cyber** [comment]
我看中文合入的是master，这个需要合到master吗？

File: docs-2.0/20.appendix/0.FAQ.md [hunk 2]
```diff
@@ -395,6 +395,19 @@ The dmp file is an error report file detailing the process exit information, whi
    ```
    $(gdb) bt
    ```
+For example:
+```bash
+$ file core.624
```
Comment (whitewum) [inline docs-2.0/20.appendix/0.FAQ.md:400]: 你这是谁打出来的core？

Comment (whitewum) [reply docs-2.0/20.appendix/0.FAQ.md:400]: 找个服务进程吧。。。人家常见的都是graph或者storage挂了

**Review by whitewum — CHANGES_REQUESTED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**